packages/@react-spectrum/list/src/ListItem.tsx

```diff
@@ -0,0 +1,60 @@
+import {classNames} from '@react-spectrum/utils';
+import {ListContext} from './List';
+import listStyles from './index.css';
+import {mergeProps} from '@react-aria/utils';
+import React, {useContext, useRef} from 'react';
+import {useFocusRing} from '@react-aria/focus';
+import {useGridCell, useGridRow} from '@react-aria/grid';
+import {useHover} from '@react-aria/interactions';
+
+
+export function ListItem(props) {
+  let {
+    item
+  } = props;
+  let {state} = useContext(ListContext);
+  let ref = useRef<HTMLDivElement>();
+  let {
+    isFocusVisible: isFocusVisibleWithin,
+    focusProps: focusWithinProps
+  } = useFocusRing({within: true});
+  let {isFocusVisible, focusProps} = useFocusRing();
+  let {hoverProps, isHovered} = useHover({});
```

Comment:
Do ListItems support being disabled?

Reply:
might be something in a later version? still waiting for spectrum direction